packages/wallets/src/features/accounts/modules/TaxInformation/components/NeedHelpMessage/NeedHelpMessage.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { InlineMessage, WalletText } from '../../../../../../components';
+
+const onClickLiveChat = () => window.LC_API.open_chat_window();
```

Comment:
i saw the same component and there you are defining `onClickLiveChat` inside the component, let be consistent, and refactor those component as it also has `onClickLiveChat` outside of itself